models/InterviewRepository.ts

```diff
@@ -0,0 +1,53 @@
+import { Interview, InterviewStatus, PrismaClient } from "@prisma/client";
+import { Interviews } from "../@types";
+
+export default class InterviewRepository {
+  prisma: PrismaClient;
+
+  constructor(prismaClient: PrismaClient) {
+    this.prisma = prismaClient;
+  }
+
+  async getInterviewById(interviewId: string): Promise<Interview> {
+    const interview = await this.prisma.interview.findUnique({ where: { id: interviewId } });
+    if (!interview) throw new Error("Error: interviewId not found");
+    return interview;
+  }
+
+  async getInterviewsByUserId(userId: string): Promise<Interviews> {
```

Comment:
I'm wondering is it necessary to add a "getInterviewsByUserIdAndStartTime" ?  If not, then there will be many interviews that does not belong to the desired time range when we load interviews for a specific week  (or is it better to filter interviews in frontend after fetching all interviews given a user?)

Reply:
We can always add more endpoints when we need it.